src/components/MovieCard/MovieCard.js

```diff
@@ -1,25 +1,22 @@
 import React from 'react'
 import { Link } from 'react-router-dom'
-import { connect } from 'react-redux'
 
-const MovieCard = (props) => {
-  const { title, movie_id, poster_path, release_date, vote_average, overview } = props.movie;
+
+const MovieCard = ({movie, user, handleDelete, handleFavorite}) => {
```

Comment:
I really like the use of destructuring the props here! User is undefined when the user tries to access their favorites because we need to pass user down when the route path is /favorites in MovieArea.

Reply:
Sounds good. I'll add that to the branch I am working on now!